### PR TITLE
Complete CRUD functionality added to Pipeline Manager class

### PR DESCRIPTION
**Pull request by gilpasternak35**

### CODE REVIEW

File: chemportal/cuchemportal/pipeline/pipeline_manager.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+    #def clone_pipeline(self, pipeline_id: int):
+        #"""Clones a Pipeline Object"""
+        # Since overriden, will call the Pipeline class deepcopy implementation which we can modify
+        #return deepcopy(self.pipelines[pipeline_id])
+
+    # Todo: now access to database
+    def publish(self, pipeline_id: int):
+        """Publishes a pipeline"""
+        self.pipelines[pipeline_id].publish()
```
Comment (rilango) [inline chemportal/cuchemportal/pipeline/pipeline_manager.py:77]: I am assuming that this is not a completed method.

File: chemportal/db/migrate/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM python:3.7-buster
```
Comment (rilango) [inline chemportal/db/migrate/Dockerfile:1]: I think, this is not required.

File: chemportal/db/migrate/migrate.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
```
Comment (rilango) [inline chemportal/db/migrate/migrate.py:1]: Please remove this one too.

File: chemportal/db/migrate/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,2 @@
+mysql-connector-python==8.0.17
```
Comment (rilango) [inline chemportal/db/migrate/requirements.txt:1]: Please remove

File: chemportal/db/setup.sql [hunk 5]
```diff
@@ -0,0 +1,96 @@
+-- MySQL Workbench Forward Engineering
```
Comment (rilango) [inline chemportal/db/setup.sql:1]: Please update the setup SQL.

File: chemportal/db/start.sh [hunk 6]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+BASE_DIR=$(dirname $0)
+
+docker stop pef_db
```
Comment (rilango) [inline chemportal/db/start.sh:5]: Please change container name to cuchem_db

File: chemportal/cuchemportal/data/configurations/portal_config.json [hunk 7]
```diff
@@ -0,0 +1,81 @@
+{
+  "id": 26, 
+  "tasks":
+  {
+  "name": "preprocess",
+  "display_name": "preprocessing_molecueles",
+  "time_created": "2021-08-02T11:22Z",
+  "implementation_spec":
```
[the remark this answers is not in the document]
Comment (gilpasternak35) [reply chemportal/cuchemportal/data/configurations/portal_config.json:8]: Not necessarily - we could take all of its attributes outside the dict. I will do this and see how the config looks upon its removal

**Review by rilango — APPROVED** [review]

**Review by rilango — APPROVED** [review]

**Pull request merged**